lib/activity.rb

```diff
@@ -0,0 +1,23 @@
+class Activity
+  attr_reader :name, :participants
+  def initialize(name)
+    @name = name
+    @participants = {}
+  end
+
+  def add_participant(name, cost)
+    @participants[name] = cost
+  end
+
+  def total_cost
+    @participants.sum{|k,v| v}
```

Comment:
k and v are *not* descriptive, you should use something that indicates what the variables are poingint to, like `name` and `cost`